editor/plugins/org.fusesource.ide.wsdl2rest.ui/src/org/fusesource/ide/wsdl2rest/ui/wizard/Wsdl2RestWizard.java

```diff
@@ -196,18 +196,72 @@ public boolean isProjectSpringBoot(IProject project) {
 		return CamelCatalogUtils.hasSpringBootDependency(projectDependencies);
 	}
 
+	
+	private org.fusesource.ide.camel.model.service.core.catalog.Dependency makeDependency(
+			String grpId, String artId, String version) {
+		org.fusesource.ide.camel.model.service.core.catalog.Dependency dep = 
+				new org.fusesource.ide.camel.model.service.core.catalog.Dependency();
+		dep.setArtifactId(artId);
+		dep.setGroupId(grpId);
+		dep.setVersion(version);
+		return dep;
+	}
+	
 	/**
-	 * Add JAX-RS dependency
+	 * Add required dependencies
 	 */
 	private void updateDependencies(IProgressMonitor monitor) {
-		List<org.fusesource.ide.camel.model.service.core.catalog.Dependency> deps = new ArrayList<>();
-		org.fusesource.ide.camel.model.service.core.catalog.Dependency one = 
-				new org.fusesource.ide.camel.model.service.core.catalog.Dependency();
-		one.setArtifactId("jboss-jaxrs-api_2.0_spec"); //$NON-NLS-1$
-		one.setGroupId("org.jboss.spec.javax.ws.rs"); //$NON-NLS-1$
-		one.setVersion("1.0.0.Final-redhat-1"); //$NON-NLS-1$
-		deps.add(one);
 		IProject project = options.getProject();
+
+		// use project to determine if we are building a spring or blueprint project
+		boolean isBlueprint = isProjectBlueprint(project);
+		boolean isSpringBoot = isProjectSpringBoot(project);
```

Comment:
I think we also have a case when it is Spring but not SpringBoot. But maybe for wsdl2rest there is nothing specific to add in this case?